src/canvas-effects/canvas-effect.js

```diff
@@ -2803,6 +2520,10 @@ export default class CanvasEffect extends PIXI.Container {
 		);
 
 		for (let animation of oneShotAnimations) {
+			if (animation.target === 'alphaFilter') {
```

Comment:
Didn't this PR remove support for `alphaFilter` in favor of just `alpha`?

Reply:
That one is actually not a mistake. 
Yes alpha filter is removed for now. 
I added this special case to improve backwards compatibility. Old animations will probably reference she alpha filter. This is just a way to not break als those old animations. 

Regarding alpha filter itself: Maybe it becomes necessary again (though I would really like it not to be) when I try and fix the vision masking stuff, but for now alpha can safely be used and will always look identical except for the case where text and a video effect are shown in the same effect section, overlap each other and have low alpha values. 
previously the image would be composed first of 100% opaque and then made transparent, meaning the text itself would always be opaque on top of the effect. 
that is not the case anymore, instead the text would also be semi transparent on top of the semi transparent effect. 

This should not really break anything. Will probably not even effect any animation out there and if it did the difference would probably be too subtle to really notice.. I thought it a worthwhile tradeoff since the benefit of doing this is essentially an up to 16x increase in performance.

Reply:
Thanks for the explanation - though, wouldn't this have to change the property target to `alpha` then? Since the `alphaFilter` is no longer a valid target?

Reply:
Wait maybe I'm missing something here. But I thought the target attribute was the by the user of the library when using the animate/loopPropery section. 
but you're right it's probably best to instead modify the section and fix the target attribute there instead.

Reply:
Okay I should have checked first. 
so the target will always be alphaFilter for existing animations but the property will be the alpha property (so previously `alphaFiltrt.alpha`. Now the target is just the sprite itself, the property stays at alpha though. So `sprite.alpha`. This should be correct